docs/security/vulnerability-disclosure-policy.mdx

```diff
@@ -9,10 +9,9 @@ description: Clerk's vulnerability disclosure policy.
 
 #### We require that all researchers:
 
-1. Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction of data during security testing.
-
-2. Perform research only within the scope set out below.
-3. Use the identified communication channels to report vulnerability information to us; an
+1. Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction of data during security testing;
+2. Perform research only within the scope set out below;
+3. Use the identified communication channels to report vulnerability information to us; and
```

Comment:
```suggestion
1. Make every effort to avoid privacy violations, degradation of user experience, disruption to production systems, and destruction of data during security testing.
2. Perform research only within the scope set out below.
3. Use the identified communication channels to report vulnerability information to us.
```

Reply:
Added. Thanks!